File: sumOfBetweenTwoIntegers/src/main/kotlin/sumOfBetweenTwoIntegers.java
class Solution {
    public long solution(int a, int b) {
        long answer = 0;
        int p = a;
        int q = b;
        if(a > b)
        {
            p = b;
            q = a;
        }
        for(int i = p; i <= q; i++)
            answer += i;


        return answer;
    }
}

File: sumOfBetweenTwoIntegers/src/main/kotlin/sumOfBetweenTwoIntegers.kt
class Solution {
    fun solution(a: Int, b: Int): Long {
        var answer: Long = 0
        val x: Int = if(a<=b) a else b
        val y: Int = if(x==a) b else a
        for(i in x..y){
            answer += i.toLong()
        }
        return answer
    }
}